README.md

```diff
@@ -1,25 +1,18 @@
-# Go API Server for openapi
+# roomservice: a microservice for generating and assigning room codes
 [![PkgGoDev](https://pkg.go.dev/badge/github.com/galactus-player/roomservice)](https://pkg.go.dev/github.com/galactus-player/roomservice)
 
-An implementation of joinable rooms
+The roomservice microservice is used in the galactus video player to create random codes which users will join.
+This microservice creates random room codes, for example `1234`, and that code will be shared with other users when they would like to join the same room.
+Once a user retrieves this code, they can use it to join a room and retrieve room information such as the videos currently in the queue, the information needed to join the room, and the people in the room.
 
 ## Overview
 This server was generated by the [openapi-generator](https://openapi-generator.tech) project.
-By using the [OpenAPI-Spec](https://github.com/OAI/OpenAPI-Specification) from a remote server, you can easily generate a server stub.  
-
-To see how to make this your own, look here:
-
-[README](https://openapi-generator.tech)
-
-- API version: 0.0.1
-- Build date: 2020-10-12T19:27:07.274-04:00[America/New_York]
-
 
 ### Running the server
 To run the server, follow these simple steps:
 
 ```
-go run main.go
+go run cmd/roomservice/main.go
 ```
 
 To run the server in a docker container
```

Comment:
Tests already show how to make calls to server, but maybe add a small snippet on how to do that here.

Reply:
Or if you're lazy just add a link to the tests.